pkg/Microsoft.Private.CoreFx.UAP/Microsoft.Private.CoreFx.UAP.pkgproj

```diff
@@ -19,101 +17,27 @@
 
     <NETStandardVersion>2.0</NETStandardVersion>
 
+    <IsFrameworkPackage>true</IsFrameworkPackage>
+
     <!-- Include symbols in package by default-->
     <IncludeSymbolsInPackage Condition="'$(IncludeSymbolsInPackage)' == ''">true</IncludeSymbolsInPackage>
   </PropertyGroup>
 
-  <Import Condition="'$(PackageTargetRuntime)' == ''" Project="$(RefBinDir)\*.props" />
-  <ItemGroup Condition="'$(PackageTargetRuntime)' == ''">
-    <!-- Include refs -->
-    <File Include="@(FileToPackage)">
-      <TargetPath Condition="'%(File.TargetPath)' == ''">ref/$(TargetFramework)</TargetPath>
-    </File>
-    <!-- force a missing file if ref build is absent -->
-    <File Include="$(RefBinDir)/MISSING_REF_BUILD" Condition="'@(FileToPackage)' == ''" />
-
-    <_buildRIDWithMetadata Include="@(BuildRID)">
-      <TargetRuntime>%(Identity)</TargetRuntime>
-      <Version>$(PackageVersion)</Version>
-    </_buildRIDWithMetadata>
-    <Dependency Include="@(_buildRIDWithMetadata->'runtime.%(Identity).$(Id)')" />
-
-    <Dependency Include="Microsoft.NETCore.Platforms">
-      <Version>$(PlatformPackageVersion)</Version>
-    </Dependency>
-  </ItemGroup>
-
-  <Import Condition="'$(PackageTargetRuntime)' != ''" Project="$(LibBinDir)\*.props" />
-  <ItemGroup Condition="'$(PackageTargetRuntime)' != ''">
-    <!-- Include lib -->
-    <File Include="@(FileToPackage)">
-      <TargetPath Condition="'%(File.TargetPath)' == ''">runtimes/$(PackageTargetRuntime)/lib/$(TargetFramework)</TargetPath>
-    </File>
-    <!-- force a missing file if lib build is absent -->
-    <File Include="$(LibBinDir)/MISSING_LIB_BUILD" Condition="'@(FileToPackage)' == ''" />
-
-    <!-- Include native -->
-    <ExcludeNative Include="$(NativeBinDir)/*.lib" />
-    <NativeFile Include="$(NativeBinDir)/*.*" Exclude="@(ExcludeNative)">
-      <TargetPath>runtimes/$(PackageTargetRuntime)/native</TargetPath>
-    </NativeFile>
-    <File Include="@(NativeFile)" />
-    <!-- force a missing file if native build is absent -->
-    <File Include="$(NativeBinDir)/MISSING_NATIVE_BUILD" Condition="'@(NativeFile)' == ''" />
-  </ItemGroup>
-
   <ItemGroup>
     <IgnoredReference Include="System.Private.CoreLib" />
     <IgnoredReference Include="Windows" />
     <IgnoredReference Include="System.Private.Interop" />
     <IgnoredReference Include="Microsoft.Win32.Registry" /> <!-- Ignore for now since it's being exclude from closure below. Issue https://github.com/dotnet/corefx/issues/15966 -->
     <IgnoredReference Include="System.IO.IsolatedStorage" /> <!-- Ignore for now since it's being exclude from closure below. Issue https://github.com/dotnet/corefx/issues/15968 -->
```

Comment:
Can be done in a separate commit, but These two items should be revised as I believe some problems have been fixed. I can take care of it as a follow up PR.